_includes/leapmicro.html

```diff
@@ -0,0 +1,11 @@
+{% capture release %}{% labeltoreleaseleapmicro %}{{ include.type }}{% endlabeltoreleaseleapmicro %}{% endcapture %}
```

Comment:
We could do 
```
{% capture release %}{% labeltorelease %}{{ include.distro }}:{{ include.type }}{% endlabeltorelease %}{% endcapture %}
```
in leapmicro.html, leapmicro_testing.html, leap.html, testing.html and legacy,html we could then adjust it to
```
{% include leap.html distro="leap" type="current" %}
```
and then handle the rest in plugin, that way we would limit the duplication here

Reply:
Ah yes good idea!